src/lib/registration/ui/base_system_registration_dialog.rb

```diff
@@ -272,15 +272,18 @@ def register_scc_option
         )
       end
 
-      # Example URL to be used in the :register_local UI
-      EXAMPLE_SMT_URL = "https://smt.example.com"
```

Comment:
Well, the idea for pre-filling the `https://smt.example.com` default is that:

- Users will clearly see what input is expected
- In many cases users will just need to update the domain name at the end of the URL and save some typing (locally we use `https://smt.suse.cz` and `https://smt.suse.de` :wink: )

But this does not work well with the editable combo box...

Maybe use the `https://smt.example.com` default only when the SLP scan does not find any server? And if something is found display just the found servers without the default which could confuse the users... :question: